docs/admin/install.rst

```diff
@@ -124,6 +124,8 @@ You will be prompted to enter the FDE passphrase set during installation.
 
 After the disk is unlocked and Qubes starts, you will be prompted to complete the initial setup. Click the Qubes OS icon.
 
+.. warning:: Make sure **Fedora** is selected as the default VM template. Otherwise, you will need to manually switch the *Work* Qube template to Fedora in order to download the SecureDrop Workstation `dom0` configuration needed to begin the install.
```

Comment:
Does this need to be a warning vs just a bullet below that says "default VM template: Fedora" (idk what it looks like actually)? I'm thinking that just letting people follow the instructions will be easier than giving them a warning here...